packages/contact-manager/__tests__/shared/contactManagerAgentLogic.ts

```diff
@@ -36,7 +37,7 @@ export default (testContext: { getAgent: () => ConfiguredAgent; setup: () => Pro
         displayName: 'default_display_name',
         contactType: {
           type: PartyTypeType.NATURAL_PERSON,
-          origin: PartyOrigin.EXTERNAL,
+          partyOrigin: PartyOrigin.EXTERNAL,
```

Comment:
origin fields were renamed because when the party & identity entities are joined into a query using typeorm I got an error of duplicate column names.